_source/providenceConfiguration/mainConfiguration/media_processing.conf.rst

```diff
@@ -1,2 +1,201 @@
 Media_processing.conf
```

Comment:
In general this is a good start, but we'll need to add more detail. I can take on this job.

Reply:
Yes, please.